WireDomain/Sources/WireDomain/Account/BackendEnvironmentStore.swift

```diff
@@ -0,0 +1,183 @@
+//
+// Wire
+// Copyright (C) 2025 Wire Swiss GmbH
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program. If not, see http://www.gnu.org/licenses/.
+//
+
+import Foundation
+import WireLogging
+import WireNetwork
+
+private let log = WireLogger(tag: "Accounts")
+
+public struct BackendEnvironmentStore {
+
+    private let directory: URL
+    private let fileManager = FileManager.default
+
+    private let encoder = JSONEncoder()
+    private let decoder = JSONDecoder()
+
+    public init(root: URL) throws {
+        self.directory = root.appendingPathComponent("AccountData")
```

Comment:
suggestion: maybe we factor this path in a helper since a lot of objects depend on the file srtucture:
CoreDataStack, AccountStore,...

Reply:
I factored it out into `AccountURLs` but didn't get `CoreDataStack` to use it just yet as doing so would make the PR grow quite a bit.